src/utils.ts

```diff
@@ -66,7 +70,7 @@ export const getFontFamilyString = ({
 }: {
   fontFamily: FontFamily;
 }) => {
-  return FONT_FAMILY[fontFamily];
+  return [FONT_FAMILY[fontFamily], WINDOWS_EMOJI_FALLBACK_FONT].join(",");
```

Comment:
I'd use template strings so that we don't do array allocation and `join` call every time.

Reply:
👍 see 8fe2644e0892096137036ec3034d6b97abf2f350